core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -19,8 +19,14 @@
  * you change or add Alluxio configuration properties.
  */
 public enum PropertyKey {
+  CONF_DIRECTORY(Name.CONF_DIRECTORY, "${alluxio.home}/conf"),
   DEBUG(Name.DEBUG, false),
+  /**
+   * @deprecated use INSTALL_HOME or WORK_DIRECTORY instead
+   */
+  @Deprecated
```

Comment:
never mind, it would only change the meaning of `alluxio.home` if we defaulted `alluxio.work.directory` to something different from `alluxio.home`. @jsimsa convinced me to keep it as `alluxio.home`